index.html

```diff
@@ -201,5 +202,58 @@ <h4>Skills</h4><br>
             </ul>
         </div> 
 </section> 
+<!--CONTACT PAGE STARTS HERE -->
+<!--FORM DESIGN STARTS HERE -->
+<section class="page">
+    <div class="gap"></div>
+    <div class="c-form">
+    <div class="f-header">
+        <h2 class="getStart">Get started</h2>
+        <p class="About">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it .</p>
+    </div>    
```

Comment:
- [ ] Kindly stick to the Figma design by correcting these `font` so them may look as much as possible as they are in the template design. :+1: 
Your design

![Screenshot from 2022-12-08 17-13-37](https://user-images.githubusercontent.com/80458199/206486669-c0237089-44d0-435c-b9a0-f00d7abfc904.png)

Figma design
![Screenshot from 2022-12-08 17-28-47](https://user-images.githubusercontent.com/80458199/206486655-dfdd380a-248c-4b9e-b27f-cad5adeaf431.png)